app/src/main/java/org/simple/clinic/util/preference/MoshiPreferenceConverter.kt

```diff
@@ -0,0 +1,20 @@
+package org.simple.clinic.util.preference
+
+import com.f2prateek.rx.preferences2.Preference
+import com.squareup.moshi.Moshi
+
+class MoshiPreferenceConverter<T>(
```

Comment:
Do you think we need to be more specific and call this `MoshiObjectPreferenceConverter`?
I don't feel strongly about this. This is up to you :)

Reply:
Yeah, makes sense. I'll change this.